cli/smartcontract/smart_contract.go

```diff
@@ -100,22 +113,18 @@ func NewCommands() []*cli.Command {
 	}
 	invokeFunctionFlags = append(invokeFunctionFlags, options.Wallet...)
 	invokeFunctionFlags = append(invokeFunctionFlags, options.RPC...)
-	deployFlags := append(invokeFunctionFlags, []cli.Flag{
-		&cli.StringFlag{
-			Name:     "in",
-			Aliases:  []string{"i"},
-			Required: true,
-			Usage:    "Input file for the smart contract (*.nef)",
-			Action:   cmdargs.EnsureNotEmpty("in"),
-		},
-		&cli.StringFlag{
-			Name:     "manifest",
-			Aliases:  []string{"m"},
-			Required: true,
-			Usage:    "Manifest input file (*.manifest.json)",
-			Action:   cmdargs.EnsureNotEmpty("manifest"),
+	deployFlags := append(invokeFunctionFlags,
+		cloneFlag(inFlag, true),
+		cloneFlag(manifestFlag, true),
+	)
+	updateFlags := append([]cli.Flag{
+		cloneFlag(inFlag, false),
+		cloneFlag(manifestFlag, false),
+		&cli.BoolFlag{
+			Name:  "strict",
+			Usage: "Disables fallback call when data is not provided",
```

Comment:
> fallback call

Need more detailed description with NEP-22 mention and `update` differences, otherwise it's not clear to the end-user what is a "fallback call".